### ISSUE TITLE
Correct configure on Ubuntu

### ISSUE THREAD

**Issue by Flashwalker** [intel/intel-lpmd#70]
Here is the correct configuration for Ubuntu:
```bash
./autogen.sh prefix=/usr --localstatedir=/var --sysconfdir=/etc
```
If you omit the `--localstatedir=/var --sysconfdir=/etc` part you will get an installation error:
```
... «/usr/etc»: No such file or directory
```

**Comment by zhang-rui** [comment]
./autogen.sh
./autogen.sh  prefix=/usr
./autogen.sh  prefix=/usr --localstatedir=/var --sysconfdir=/etc

All of the three above works for me with Ubuntu 24.04

**Comment by Flashwalker** [comment]
@zhang-rui 
In my case, if I pass only: `./autogen.sh prefix=/usr`, the Makefile will generate:
```
...
sysconfdir = /usr/etc
localstatedir = /usr/var
lpmd_confdir = /usr/etc/intel_lpmd
lpmd_rundir = /usr/var/run/intel_lpmd
...
```
Which are the wrong paths.

I am installing from the Releases archive.

**Comment by zhang-rui** [comment]
right. In my case, although the program doesn't break, it is true that these files are all put into /usr, while they should better stay at /var/run or /etc.

but this is not a problem for the intel-lpmd tool, right? we should ask Ubuntu to use "./autogen.sh prefix=/usr --localstatedir=/var --sysconfdir=/etc" for intel_lpmd configure and install, right?
